src/script/Root.tsx

```diff
@@ -20,14 +20,16 @@
 import {Global} from '@emotion/core';
 import {COLOR, FlexBox, Loading, StyledApp} from '@wireapp/react-ui-kit';
 import createBrowserHistory from 'history/createBrowserHistory';
-import React, {lazy, Suspense} from 'react';
+import React, {lazy, Suspense, useEffect} from 'react';
 import {Redirect, Route, Router, Switch} from 'react-router-dom';
 import {ROUTE} from 'script/route';
 
 const history = createBrowserHistory();
 
 interface Props {}
 
+const LANG_QUERY_KEY = 'hl';
```

Comment:
maybe should add this to QUERY_KEYs enum

Reply:
done 😄